src/jtagmkI.c

```diff
@@ -1192,9 +1224,9 @@ static void jtagmkI_print_parms1(const PROGRAMMER *pgm, const char *p, FILE *fp)
   }
 
   if (pgm->extra_features & HAS_VTARG_READ) {
-    if (jtagmkI_getparm(pgm, PARM_OCD_VTARGET, &vtarget) < 0)
+    if (jtagmkI_get_vtarget(pgm, &vtarget) < 0)
       return;
-    fmsg_out(fp, "%sVtarget               : %.1f V\n", p, 6.25 * (unsigned)vtarget / 255.0);
+    fmsg_out(fp, "%sVtarget       : %.1f V\n", p, vtarget);
```

Comment:
There are still issues with the number of spaces here; 8 spaces missing.

Reply:
jtagmkI is done, also for the two `ICE HW version` lines below.